test/AutoinstSoftware_test.rb

```diff
@@ -3,16 +3,16 @@
 require_relative "test_helper"
 
 # storage-ng
-=begin
 Yast.import "AutoinstSoftware"
+=begin
 Yast.import "AutoinstData"
 Yast.import "Profile"
 =end
 
 describe "Yast::AutoinstSoftware" do
   # storage-ng
   before :all do
-    skip("pending of storage-ng")
+#    skip("pending of storage-ng")
```

Comment:
If this skip call can be removed, it's time to remove the whole `before` block.

Reply:
I wanted to see if it pass even in travis. If yes, I'll adapt the test and keep it.